test/e2e/cws-tests/tests/test_e2e_cws_docker.py

```diff
@@ -126,6 +126,13 @@ def test_open_signal(self):
                 self.fail("check ruleset_loaded timeouted")
             self.app.check_for_ignored_policies(self, attributes)
 
+        with Step(msg="check self_tests", emoji=":test_tube:"):
+            rule_id = "self_test"
+            event = self.app.wait_app_log(f"rule_id:{rule_id}")
+            attributes = event["data"][0]["attributes"]["attributes"]
+            self.assertEqual(rule_id, attributes["agent"]["rule_id"], "unable to find rule_id tag attribute")
+            self.assertTrue("failed_tests" not in attributes, "self_tests have failed")
```

Comment:
nit but maybe we could print the list of tests that failed in the assertion message to ease debugging